ConfigCreator/ConfigCreator/Example.json

```diff
@@ -0,0 +1,44 @@
+{
+  "icon": "",
+  "background": "",
+  "updatePath": "",
+  "version": "V1.0",
+  "generation": 2,
+  "instanceName": "",
```

Comment:
Warning: the instanceName should be filled out, otherwise it will lead to WindowError, because the file name of the generated json file is based on this. Not tested if Non-ASCII letter can pass.